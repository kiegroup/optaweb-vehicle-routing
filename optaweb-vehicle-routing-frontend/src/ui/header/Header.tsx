import { Brand, PageHeader } from '@patternfly/react-core';
import * as React from 'react';
import NavigationWithRouter from './Navigation';

const Header: React.FC = () => (
  <PageHeader
    logo={<Brand src="./assets/images/optaPlannerLogoDarkBackground200px.png" alt="OptaPlanner Logo" />}
    topNav={<NavigationWithRouter />}
  />
);

export default Header;
