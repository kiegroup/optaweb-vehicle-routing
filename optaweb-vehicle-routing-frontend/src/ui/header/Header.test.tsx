import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import * as React from 'react';
import Header from './Header';

describe('Header component', () => {
  it('should match snapshot', () => {
    const header = shallow(<Header />);
    expect(toJson(header)).toMatchSnapshot();
  });
});
