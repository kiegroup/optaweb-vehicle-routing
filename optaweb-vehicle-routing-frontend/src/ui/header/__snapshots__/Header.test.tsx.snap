// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<PageHeader
  logo={
    <Brand
      alt="OptaPlanner Logo"
      src="./assets/images/optaPlannerLogoDarkBackground200px.png"
    />
  }
  topNav={<withRouter(Navigation) />}
/>
`;
